modules/nw-egress-ips-namespaces.adoc

```diff
@@ -0,0 +1,68 @@
+// Module filename: nw-egress-ips-namespaces.adoc
+//
+// Module included in the following assemblies:
+// * networking/managing-networking.adoc
+
+[id='nw-egress-ips-namespaces-{context}']
+= Enabling Automatic Egress IPs
+
+As a cluster administrator, you can assign
+egress IP addresses to namespaces by setting the `egressIPs` field to the
+`NetNamespace` resource.
+
+With fully automatic egress IPs, you can set the `egressCIDRs` field
+of each node's `HostSubnet` resource to indicate the range of egress
+IP addresses that can be hosted.
+Namespaces that have requested egress IP addresses are matched
+with nodes that are able to host those egress IP addresses, then the egress IP
+addresses are assigned to those nodes.
+
+High availability is automatic. If a node hosting egress IP addresses
+goes down and there are nodes that are able to host those egress IP addresses,
+based on the `egressCIDR` values of the `HostSubnet` resources, then the egress
+IP addresses move to a new node. When the original egress IP address node
+comes back online, the egress IP addresses automatically move to balance
+egress IP addresses across nodes.
+
+[IMPORTANT]
+====
+You cannot use manually assigned and automatically assigned egress IP addresses
+on the same nodes or with the same IP address ranges.
+====
+
+.Procedure
+
+. Update the `NetNamespace` with the egress IP address:
++
+----
+ $ oc patch netnamespace <project_name> -p '{"egressIPs": ["<IP_address>"]}'
+----
++
+For example, to assign `project1` to an IP address of 192.168.1.100 and `project2`
+to an IP address of 192.168.1.101:
++
+----
+$ oc patch netnamespace project1 -p '{"egressIPs": ["192.168.1.100"]}'
+$ oc patch netnamespace project2 -p '{"egressIPs": ["192.168.1.101"]}''
+----
+
+. Indicate which nodes can host egress IP addresses by
+setting their `egressCIDRs` fields:
++
+----
+$ oc patch hostsubnet <node_name> -p \
+  '{"egressCIDRs": ["<IP_address_range_1>", "<IP_address_range_2>"]}'
+----
++
+For example, to set `node1` and `node2` to host egress IP addresses
+in the range 192.168.1.0 to 192.168.1.255:
++
+----
+$ oc patch hostsubnet node1 -p '{"egressCIDRs": ["192.168.1.0/24"]}'
+$ oc patch hostsubnet node2 -p '{"egressCIDRs": ["192.168.1.0/24"]}'
+----
++
+{product-title} automatically assigns specific egress IP addresses to available
+nodes, in a balanced way. In the example above, it assigns the egress
```

Comment:
s/nodes,/nodes

Not above.

Reply:
Applied in https://github.com/openshift/openshift-docs/pull/14555.